pkg/install/templates/memcache-dep.yaml.tmpl

```diff
@@ -18,6 +18,7 @@ spec:
     spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
+      serviceAccountName: flux
```

Comment:
Do you need an explicit account to make it work on openshift? Flux needs permissions Memecached doesn't, so, the default account is a better choice.

Sorry for not writing this before.

Reply:
Previously I had always tested it with an explicit service account. But testing it just now it doesn't need it. I will remove it and push a change now.